<!DOCTYPE HTML>
<html lang="en">

<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

    <title>Gunjan Baid</title>

    <meta name="author" content="Gunjan Baid">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <link rel="stylesheet" type="text/css" href="stylesheet.css">
    <link rel="icon" href="images/GunjanBaid.jpg">
</head>

<body>
    <table
        style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
        <tbody>
            <tr style="padding:0px">
                <td style="padding:0px">
                    <table
                        style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                        <tbody>
                            <tr style="padding:0px">
                                <td style="padding:2.5%;width:63%;vertical-align:middle">
                                    <p style="text-align:center">
                                        <name>Gunjan Baid</name>
                                    </p>
                                    <p>I'm a software engineer based in San Francisco, CA. My research interests include
                                        applications of machine learning to problems in health and genomics. Previously,
                                        I was at <a href="https://research.google">Google Research</a>
                                        where I worked on genomics and multimodal machine learning. I
                                        completed my Masters and
                                        undergraduate studies at UC Berkeley and was a part of the RISELab.
                                    </p>
                                    <p>
                                        At Google, I worked on <a href="
                                            https://doi.org/10.1038/s41587-022-01435-7">DeepConsensus</a> for
                                        error correction in PacBio sequencing
                                        data and <a href="https://github.com/google/deepvariant">DeepVariant</a> for
                                        small variant calling. My work in multimodal includes text-to-image generation
                                        with <a href="https://parti.research.google/">Parti</a> and <a
                                            href="https://blog.google/products/search/multisearch/">Multisearch</a> for
                                        Google Lens.
                                    </p>
                                    <p style="text-align:center">
                                        <a href="mailto:gunjanbaid6@gmail.com">Email</a> &nbsp/&nbsp
                                        <a href="https://scholar.google.com/citations?hl=en&user=6vV6af0AAAAJ">Google
                                            Scholar</a>
                                        &nbsp/&nbsp
                                        <a href="https://twitter.com/gunjan_baid">Twitter</a>
                                        &nbsp/&nbsp
                                        <a href="https://github.com/gunjanbaid/">GitHub</a>
                                    </p>
                                </td>
                                <td style="padding:2.5%;width:40%;max-width:40%">
                                    <a href="images/GunjanBaid.jpg"><img style="width:100%;max-width:100%"
                                            alt="profile photo" src="images/GunjanBaid.jpg" class="hoverZoomLink"></a>
                                </td>
                            </tr>
                        </tbody>
                    </table>
                    <table
                        style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                        <tbody>
                            <tr>
                                <td style="padding:20px;width:100%;vertical-align:middle">
                                    <heading>Research</heading>
                                </td>
                            </tr>
                        </tbody>
                    </table>

                    <table
                        style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                        <tbody>

                            <!-- -------------------------------------------- PAPERS -------------------------------------------- -->
                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/deepconsensus.png' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>DeepConsensus: improving accuracy of genomic sequencing</papertitle>
                                    <br>
                                    <em>Nature Biotechnology</em>, 2022
                                    <br>
                                    <a href="https://doi.org/10.1038/s41587-022-01435-7">Paper</a>
                                    /
                                    <a
                                        href="https://blog.google/technology/health/a-new-genome-sequencing-tool-powered-with-our-technology">Blog
                                        Post</a>
                                    /
                                    <a href="https://github.com/google/deepconsensus">GitHub</a>
                                    <p></p>
                                    <p>
                                        DeepConsensus uses a transformer model to reduce in PacBio
                                        Circular Consensus Sequencing (CCS) data by 59%. This method is now deployed on
                                        PacBio <a href="https://www.pacb.com/revio/ ">Revio</a> sequencers. Awarded
                                        Best Scientific Breakthrough (1 project across Alphabet).
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/parti.png' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>
                                        Scaling autoregressive models for content-rich text-to-image generation
                                    </papertitle>
                                    <br>
                                    <em>TMLR</em>, 2022
                                    <br>
                                    <a href="https://doi.org/10.48550/arXiv.2206.10789">Paper</a>
                                    /
                                    <a href="https://parti.research.google/">Website</a> /
                                    <a
                                        href="https://blog.google/technology/research/how-ai-creates-photorealistic-images-from-text/">Blog
                                        Post</a>
                                    /
                                    <a href="https://github.com/google-research/parti">GitHub</a>
                                    <p></p>
                                    <p>
                                        Parti (Pathways Autoregressive Text-to-Image) is an autoregressive
                                        text-to-image generation model that achieves high-fidelity photorealistic image
                                        generation and supports content-rich synthesis involving complex compositions
                                        and world knowledge.
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <center><img src='images/pfda.jpg' width="80%"></center>
                                    </div>
                                    <!-- <div>
                                        <img src='images/pfda.png' width="100%">
                                    </div>                                     -->
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>PrecisionFDA Truth
                                        Challenge
                                        V2: Calling variants from short and long reads in difficult-to-map
                                        regions</papertitle>
                                    <br>
                                    <em>Cell Genomics</em>, 2022
                                    <br>
                                    <a href="https://doi.org/10.1016/j.xgen.2022.100129">Paper</a>
                                    /
                                    <a href="https://precision.fda.gov/challenges/10/results">Challenge Results</a>
                                    /
                                    <a
                                        href="https://ai.googleblog.com/2020/09/improving-accuracy-of-genomic-analysis.html">Blog
                                        Post</a>
                                    <p></p>
                                    <p>
                                        This challenge assessed variant calling pipelines on reference datasets, with a
                                        focus on difficult-to-map regions, segmental duplications, and the Major
                                        Histocompatibility Complex (MHC). DeepVariant recevied
                                        best overall accuracy in 3 of 4 sequencing instrument categories, and we
                                        released an improved version of the submitted model as DeepVariant v1.0.
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/nanopore.webp' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>
                                        Accelerated identification of disease-causing variants with ultra-rapid nanopore
                                        genome sequencing
                                    </papertitle>
                                    <br>
                                    <em>Nature Biotechnology</em>, 2022
                                    <br>
                                    <a href="https://doi.org/10.1038/s41587-022-01221-5">Paper</a>
                                    /
                                    <a
                                        href="https://blogs.nvidia.com/blog/2022/02/18/guinness-world-record-fastest-dna-sequencing/">Blog
                                        Post</a>
                                    <p></p>
                                    <p>
                                        Whole-genome sequencing was performed for twelve patients in critical care using
                                        an accelerated pipeline. This pipeline performs distributed sequencing, near
                                        real-time base calling and alignment, accelerated variant calling, and custom
                                        variant prioritization. We received a <a
                                            href="https://www.guinnessworldrecords.com/world-records/675050-fastest%C2%A0dna-sequencing%C2%A0technique">Guiness
                                            World Record</a> for fastest DNA sequencing technique.
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/nanopore-pipeline.png' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>
                                        Ultrarapid nanopore genome sequencing in a critical care setting
                                    </papertitle>
                                    <br>
                                    <em>New England Journal of Medicine</em>, 2022
                                    <br>
                                    <a href="https://doi.org/10.1056/NEJMc2112090">Paper</a>
                                    <p></p>
                                    <p>
                                        Whole-genome sequencing was performed for twelve patients in critical care using
                                        an accelerated pipeline. The shortest time from arrival of the blood sample to
                                        the initial diagnosis was 7 hours 18 minutes. A pathogenic or
                                        likely pathogenic variant was identified in five out of 12 patients.
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/pepper.png' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>Haplotype-aware variant
                                        calling with
                                        PEPPER-Margin-DeepVariant enables high accuracy in nanopore
                                        long-reads</papertitle>
                                    <br>
                                    <em>Nature Methods</em>, 2021
                                    <br>
                                    <a href="https://doi.org/10.1038/s41592-021-01299-w">Paper</a>
                                    /
                                    <a href="https://github.com/kishwarshafin/pepper">GitHub</a>
                                    <p></p>
                                    <p>
                                        PEPPER-Margin-DeepVariant is a haplotype-aware variant calling pipeline for
                                        Oxford Nanopore data that produces state-of-the-art results.
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/deeptrio.png' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>
                                        DeepTrio: variant calling in families using deep learning
                                    </papertitle>
                                    <br>
                                    <em>bioRxiv</em>, 2021
                                    <br>
                                    <a href="https://doi.org/10.1101/2021.04.05.438434">Paper</a>
                                    /
                                    <a
                                        href="https://opensource.googleblog.com/2021/04/analyzing-genomic-data-in-families-with-deep-learning.html">Blog
                                        Post</a>
                                    <p></p>
                                    <p>
                                        We extend DeepVariant to to consider the genetic variants in mother-father-child
                                        trios. Coexamination of available parental data improves the accuracy of variant
                                        calls for the child. The DeepTrio model was released as part of DeepVariant
                                        v1.1.
                                    </p>
                                </td>
                            </tr>

                            <tr>
                                <td style="padding:20px;width:35%;vertical-align:middle">
                                    <div>
                                        <img src='images/dataset.png' width="100%">
                                    </div>
                                </td>
                                <td style="padding:20px;width:65%;vertical-align:top">
                                    <papertitle>
                                        An extensive sequence dataset of gold-standard samples for benchmarking and
                                        development
                                    </papertitle>
                                    <br>
                                    <em>bioRxiv</em>, 2020
                                    <br>
                                    <a href="https://doi.org/10.1101/2020.12.11.422022">Paper</a>
                                    /
                                    <a
                                        href="https://console.cloud.google.com/storage/browser/brain-genomics-public/research/sequencing/">Dataset</a>
                                    <p></p>
                                    <p>
                                        We provide a benchmark dataset of WGS and WES samples from HG001-7 and NA12878/9
                                        from a variety of sequencers. We characterize properties of the data and present
                                        variant calling results on these samples.
                                    </p>
                                </td>
                            </tr>

                        </tbody>
                    </table>

                    <table
                        style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                        <tbody>
                            <tr>
                                <td style="padding:0px">
                                    <br>
                                    <p style="text-align:center;font-size:small;">
                                        Design and source code from <a href="https://jonbarron.info/">Jon Barron's
                                            website</a>.
                                    </p>
                                </td>
                            </tr>
                        </tbody>
                    </table>
                </td>
            </tr>
    </table>
</body>

</html>